README.md

```diff
@@ -31,3 +31,78 @@ There are capistrano tasks to connect to the Solr UI for managing solr that can
 
 * Would it be better to go from a server on Princeton's network (e.g., the bibdata-staging worker machine), so that Developer's internet connection doesn't affect benchmarking
```

Comment:
@maxkadel even though this is not in your PR can you edit so that it is a statement and not a question. 'It would be better...'

Reply:
Good catch. I'm tempted to take out that whole section, since we now have the "loadtest" server - what do you think @christinach?